sandbox/routes/patient/index.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+let routes = [];
+
+/* 
+    Imports all of the routes from the files in the directory
+*/
+fs.readdirSync(__dirname)
```

Comment:
While this is quite clever it would be more readable if this was just a list of imports - there aren't that many to import and we're not likely to add a lot more.